### ISSUE TITLE
category case

### ISSUE THREAD

**Issue by porjo** [spf13/hugo#764]
Specifying a category with an upper-case first letter (e.g. `categories = ["Development"]`) produces an upper-case category link on the post page. Following that link produces a 404 error.

Either a) the link be auto lower-cased or b) both cases should work

**Comment by bjornerik** [comment]
Can you be a little more specific about what "category link on the post page" you are talking about?

**Comment by porjo** [comment]
Here's an example blog post: http://blog.mapstrata.com/post/AngularJS_+_Martini_html5mode/

At the top you will see *Under Development* - clicking on "Development" results in 404. The post header code is:
```ini
+++
date = "2014-03-26T06:33:52Z"
title = "AngularJS + Martini: html5mode"
tags = [ "AngularJS", "Go", "javascript" ]
categories = [ "Development" ]
+++
```

Changing to `categories = [ "development" ]` fixes the 404.

**Comment by bjornerik** [comment]
OK, without seeing your actual template -- my best guess is you are building this url yourself by concatenating your "baseUrl + category '

I'm not sure that's something Hugo should/could fix.

You can easily fix this by using the lower in your template: `{{ $myCategory | lower  }}`

**Comment by porjo** [comment]
Thanks, using `| lower` worked. 

I'm a little confused however, as the theme ([purehugo](https://github.com/dplesca/purehugo)) already uses `| urlize` in the [same location](https://github.com/dplesca/purehugo/blob/master/layouts/_default/single.html#L22), and AFAIK that should be making the link lower-case!?

**Comment by porjo** [comment]
Ah, looks like `Urlize` was updated after v0.12 (which I'm using) to lowercase links.

Time for a new release! :smile:

**Comment by bjornerik** [comment]
Urlize should and does (at least now) lower-case the url.

What version of Hugo are you using? (the output of "hugo version").

I see this was fixed in #491 ?

OK, closing this.
